Superpowered/OpenSource/SuperpoweredIOSAudioIO.mm

```diff
@@ -113,9 +113,31 @@ - (id)initWithDelegate:(NSObject<SuperpoweredIOSAudioIODelegate> *)d
 #if (USES_AUDIO_INPUT == 1)
```

Comment:
This is an important file to review @tehsven @cwalo

Reply:
lol
<img width="5120" height="1860" alt="Screenshot 2025-07-15 at 10 36 28" src="https://github.com/user-attachments/assets/3510c9f8-fd35-468d-9b55-cc0aab552b2b" />

Reply:
@JazariMusic pushed a fix for this: https://github.com/ResonantCavity/Low-Latency-Android-Audio-iOS-Audio-Engine/compare/22b5ab2aa695...d8ca748dac62